packages/block-editor/src/components/block-list/block.js

```diff
@@ -242,14 +242,83 @@ function BlockListBlock( {
 		className
 	);
 
-	// Determine whether the block has props to apply to the wrapper.
-	if ( blockType.getEditWrapperProps ) {
-		wrapperProps = {
-			...wrapperProps,
-			...blockType.getEditWrapperProps( attributes ),
-		};
-	}
 	const blockElementId = `block-${ clientId }`;
+	const Animated = animated[ tagName ];
+
+	return (
+		<Animated
+			// Overrideable props.
+			aria-label={ blockLabel }
+			role="group"
+			{ ...props }
+			id={ blockElementId }
+			ref={ ref }
+			className={ classnames( wrapperClassName, props.className ) }
+			data-block={ clientId }
+			data-type={ name }
+			// Only allow shortcuts when a blocks is selected and not locked.
+			onKeyDown={ isSelected && ! isLocked ? onKeyDown : undefined }
+			// Only allow selection to be started from a selected block.
+			onMouseLeave={ isSelected ? onMouseLeave : undefined }
+			tabIndex="0"
+			{ ...wrapperProps }
+			style={ {
+				...( ( wrapperProps && wrapperProps.style ) || {} ),
+				...( props.style || {} ),
+				...animationStyle,
+			} }
+		>
+			{ children }
+		</Animated>
+	);
+} );
+
+const elements = [ 'p', 'div' ];
```

Comment:
To do: add list of all block level element.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements